cmd/autoowners/main.go

```diff
@@ -294,18 +326,25 @@ func pullOwners(directory string, whitelist []string) ([]string, error) {
 
 	config := filepath.Join(operatorRoot, "config")
 	templates := filepath.Join(operatorRoot, "templates")
-	for _, orgRepo := range orgRepos {
-		logrus.WithField("orgRepo", fmt.Sprintf("%+v", *orgRepo)).Info("handling repo ...")
+	for i, orgRepo := range orgRepos {
+		logrus.WithField("index", i).
```

Comment:
I logging a loop index actually useful on `Info` level?

Reply:
The `Info` is for the `repo handling ...`  and `index` is a bonus there (no other cost to output it).
I wound like to keep this.

Reply:
The cost is the additional clutter in the output, but /shrug, not going to fight over this.

Reply:
Change it back since you insist.